label_studio/tasks/api.py

```diff
@@ -369,7 +369,11 @@ def get_queryset(self):
         return Annotation.objects.filter(Q(task=task) & Q(was_cancelled=False)).order_by('pk')
 
     def delete_draft(self, draft_id, annotation_id):
-        return AnnotationDraft.objects.filter(id=draft_id).delete()
+        try:
+            draft = AnnotationDraft.objects.get(id=draft_id)
+            draft.delete()
+        except AnnotationDraft.DoesNotExist:
+            pass
```

Comment:
Could you add a comment explaining why we need to call delete on the draft object (ie, AnnotationDraft#delete which updates `created_labels_drafts` won't be called if we call delete on the queryset)

Reply:
Additionally, I think it would be helpful to clarify that _task_ drafts are typically deleted here, that is, AnnotationDraft objects where `annotation` is `None` - which is why this code: https://github.com/HumanSignal/label-studio/blob/e991893f78c4d43ed3df1039bb75c2f9dc605a74/label_studio/tasks/models.py#L1032-L1039 doesn't delete the annotation drafts in question.

Reply:
Added comment.